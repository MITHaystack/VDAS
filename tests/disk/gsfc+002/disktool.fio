[disk0]
rw=write
iodepth=8
directory=/mnt/disk0
write_bw_log=disk0_write
fill_device=1
stonewall

[disk1]
rw=write
iodepth=8
directory=/mnt/disk1
write_bw_log=disk1_write
fill_device=1
stonewall

[disk2]
rw=write
iodepth=8
directory=/mnt/disk2
write_bw_log=disk2_write
fill_device=1
stonewall

[disk3]
rw=write
iodepth=8
directory=/mnt/disk3
write_bw_log=disk3_write
fill_device=1
stonewall
