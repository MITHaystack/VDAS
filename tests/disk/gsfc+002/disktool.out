
fio: terminating on signal 2
disk0;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;7687095736;504105;15614946;0;0;0.000000;0.000000;2;321833;7.603940;388.733988;94664;1303856;100.169758%;493125.700476;136240.171134;9.283017%;36.103541%;2114942;0;759;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;77.62%;22.23%;0.07%;0.02%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.02%;0.01%;0.01%;0.01%;0.01%;0.01%;0.00%;0.00%;0.00%;0.00%

disk1;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;7687095668;504407;15605604;0;0;0.000000;0.000000;2;422835;7.590629;398.566382;46282;786304;100.221324%;493675.210347;137252.538917;9.392594%;36.881619%;2115813;0;762;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;75.53%;24.32%;0.07%;0.02%;0.02%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.00%;0.00%;0.00%;0.00%

disk2;2;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;7687095704;488055;16128467;0;0;0.000000;0.000000;2;822049;7.862574;422.135591;28054;808000;100.321677%;478149.163970;142917.973857;9.052560%;36.093222%;2165655;0;779;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;74.23%;25.63%;0.07%;0.03%;0.02%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.00%;0.00%

disk3;3;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;1375569580;541049;2603428;0;0;0.000000;0.000000;2;126738;7.051834;359.747006;201648;783744;99.983790%;528282.353425;148685.019867;9.906020%;38.388094%;383102;0;177;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;78.60%;21.27%;0.06%;0.02%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.00%;0.00%;0.00%;0.00%;0.00%

